tests/data/fmtonoff.py

```diff
@@ -150,7 +150,7 @@ def single_literal_yapf_disable():
     BAZ = {
         (1, 2, 3, 4),
         (5, 6, 7, 8),
-        (9, 10, 11, 12),
+        (9, 10, 11, 12)
     }  # yapf: disable
```

Comment:
we want trailing comma here too?

Reply:
You're commenting on the input side. To preserve the spirit of the test, I had to make a change to the input so that things would still get formatted.

Reply:
Ah great then!